src/common/camera.h

```diff
@@ -16,12 +16,17 @@
 
 class camera {
   public:
-    void initialize(double aspect_ratio = 1.0) {
+    void initialize(double image_width, double image_height) {
```

Comment:
Do we want to set parameter values for image_width and image_height?

I think no

Reply:
I'm not sure I understand — are you raising the possibility of default values? If so, I'd also fall back to no.

Reply:
Sorry, yes, default values. So no.